platform-server-kernel/src/main/resources/mail/registration-confirmation.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:th="http://www.thymeleaf.org"
+      xmlns:cod="http://cod.uno"
+      xmlns:layout="http://www.thymeleaf.org/xsd/thymeleaf-extras-dialect-2.1.xsd"
+      layout:decorator="layout/main">
+<head>
+    <title>Coduno Email Notification Service</title>
+</head>
+
+<body>
+<div id="textcontent" layout:fragment="textcontent">
+
+    This is the confirmation-email - you a are now registered to the challenge <b><span th:text="${challengeName}">Coduno</span></b>.
```

Comment:
No need to explicitly state that "This is the confirmation-email". Just cut that part and start with "You are now ...".